src/app/technical-record/ministry-plates-template/ministry-plates-template.component.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  Component,
+  OnInit,
+  ChangeDetectionStrategy,
+  Input,
+  Output,
+  EventEmitter,
+  OnDestroy
+} from '@angular/core';
+import { PLATE_ISSUE_VALUES } from '@app/app.enums';
+import { Applicant, Plate } from '@app/models/tech-record.model';
+import { FormControl, FormGroup } from '@angular/forms';
+import { UserService } from '@app/app-user.service';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'vtm-ministry-plates-template',
+  templateUrl: './ministry-plates-template.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class MinistryPlatesTemplateComponent implements OnInit, OnDestroy {
+  @Input() applicantDetails: Applicant;
+  @Output() plateHandler = new EventEmitter<Plate>();
+  newPlate: Plate = {} as Plate;
+  platesForm: FormGroup;
+  issueValues = Object.values(PLATE_ISSUE_VALUES);
+  platesFormSubscription: Subscription;
+
+  constructor(private loggedUser: UserService) {}
+
+  ngOnInit() {
+    this.applicantDetails = !!this.applicantDetails ? this.applicantDetails : ({} as Applicant);
+
+    this.platesForm = new FormGroup({});
+    this.platesForm.addControl('plateReasonForIssue', new FormControl());
+
+    this.platesFormSubscription = this.platesForm.valueChanges.subscribe((result) => {
+      const plateReason = result.plateReasonForIssue.includes('original')
+        ? 'Original'
+        : 'Replacement';
+      this.newPlate = this.setPlateAttributes(plateReason);
+      this.plateHandler.emit(this.newPlate);
+    });
```

Comment:
not sure you need `valueChanges` to grap the form data, You can grap all form update for creating the plate object when the user presses 'send ministry plate' and dialog is closed